native/eu_cqse_clang_ClangBinding.cpp

```diff
@@ -39,6 +40,34 @@ struct ClangBindingVisitorParameter {
     return; \
```

Comment:
I think we could move the `return 0` and `return` out of the macro and avoid the code duplication.

```
CLANG_JNI_BEGIN_EXCEPTION_HANDLER()
return some_throwing_call();
CLANG_JNI_END_EXCEPTION_HANDLER("foo")
return 0
```

Also, you may consider using `__func__` in the macro rather than an explicit string parameter.